core/common/src/thrift/file_system_master.thrift

```diff
@@ -124,6 +124,9 @@ struct TAcl {
   1: optional string owner
   2: optional string owningGroup
   3: optional list<TAclEntry> entries
+  4: optional i16 mode
+  5: optional bool isDefault
+  6: optional bool isEmpty
```

Comment:
Is this just for defaults? If so, it should be something like isDefaultEmpty

Reply:
done